frontend/playground-frontend/src/features/Train/features/Tabular/components/TabularReviewStep.tsx

```diff
@@ -27,16 +36,32 @@ const TabularReviewStep = ({
       <Typography>{`Is Default?: ${trainspace.datasetData.isDefaultDataset}`}</Typography>
       <Typography>{`Target Column: ${trainspace.parameterData.targetCol}`}</Typography>
       <Typography>{`Feature Columns: ${trainspace.parameterData.features.join()}`}</Typography>
+      <Controller
+        control={control}
+        name="notificationPhoneNumber"
+        render={({ field: { onChange } }) => (
+          <TextField label="Phone Number" onChange={onChange} />
```

Comment:
@Dashrekker do you have any validation mechanisms in place to ensure that a phone number that's entered is valid? Similar thing for email? Might be good to validate this before submitting training request